deploy/olm/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 The Jetstack cert-manager contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.11 as chart-get
+
+RUN apk add wget tar
+RUN wget https://charts.jetstack.io/charts/cert-manager-v0.15.0-alpha.0.tgz
+# TODO: add some kind of verification
+RUN tar xzf cert-manager-v0.15.0-alpha.0.tgz
```

Comment:
Given that we are naming version numbers here, it might be better for us to have these files in a completely separate repo altogether to avoid lock-step version bumping issues like we had before. With the way we're now building these images it doesn't really make sense anymore to have these included here IMO, it creates a 'circular dependency' conceptually